_posts/2021-08-25-debezium-1-7-beta1-released.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Debezium 1.7.0.Beta1 Released
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, kafka ]
+author: rkerner
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
+
+This release brings https://docs.nats.io/developing-with-nats-streaming/streaming#nats-streaming-overview[NATS Streaming]
+support for Debezium Server along with many other fixes and enhancements. +
+Also this release is the first one tested with Apache Kafka 2.8.
+
++++<!-- more -->+++
+
+Also this release has a breaking change for the MySQL Connector. +
+The MySQL driver was updated to the latest version 8.0.26 with https://issues.jboss.org/browse/DBZ-3833[DBZ-3833].
+This update comes with a new timezone handling and configuration options. +
+Detailed information can be found in the https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-datetime-types-processing.html[MySQL docs].
+
+This month we added the https://debezium.io/documentation/reference/operations/debezium-ui.html[Debezium UI]
+to our regular release process. If you want to learn more about the Debezium UI have a look at our
+recent link:/blog/2021/08/12/introducing-debezium-ui/[release announcement].
+
+Overall, https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12317320&version=12354171[81 issues] were fixed for this release. +
```

Comment:
We usually mention a few key issues that were fixed. E.g. see here ("Further Fixes..."): https://debezium.io/blog/2021/06/10/debezium-1-6-beta2-released/. Perhaps something on asorted Oracle connector fixes?

Reply:
I have no idea about any issue that's worth listing here and I have no idea what I could write about the Oracle issues in this release.